### ISSUE TITLE
iOS 12 / Swift 4.2: IQKeyboardManager causes UITableView to shift down when filtering results from UISearchBar

### ISSUE THREAD

**Issue by mbain** [hackiftekhar/IQKeyboardManager#1511]
**Describe the bug**
Our app is being written in Swift 4.2, and run on iOS 12.  When IQKeyboardManager is enabled in the App Delegate, if we go to a view controller with a search bar above a table view, as soon as a user taps in the search field and the keyboard appears, it shifts the UITableView down, showing a black bar.  This black bar is coming from the UIWindow view being exposed.

**To Reproduce**
1. Create an app for iOS 12.  Enable IQKeyboardManager in the App Delegate.
2. Create a ViewController that has a UISearchBar at the top (ours is added to a UINavigationController), and a UITableView laid-out below.
3. Launch the app, tap in the search field.  You will observe the keyboard being displayed, and the UITableView being shifted down, exposing a horizontal black bar.
4. If you explode the view hierarchy in the Xcode debugger, you will see that the black is coming from UIWindow.

**Expected behavior**
When IQKeyboardManager is disabled, the top of the UITableView remains pinned to the bottom of the UISearchBar / UINavigationBar.  This behavior should be consistent with IQKeyboardManager enabled.  In this sort of view, it is not necessary to specify layout constraints manually.

**Screenshots**
Before tapping in the search field:
![img_7941](https://user-images.githubusercontent.com/14931151/51008443-a8577d00-1500-11e9-9aad-0ff213385d95.jpg)

After tapping in the search field:
![img_7942](https://user-images.githubusercontent.com/14931151/51008442-a8577d00-1500-11e9-98a0-1ff17a2fb493.jpg)

**Demo Project**
Not available.

### Versions
**Xcode:** 10.1
**Mac OS:** 10.14.2
**Simulator/Device:** Device: iPhone 7
**Simulator/Device Name:** Device: iPhone X
**Library Version:** 6.2.0

**Additional context**
Tested on multiple devices and multiple simulators.  The behavior is 100% consistent.

**Comment by hackiftekhar** [comment]
okay, I'll try to reproduce the issue.

**Comment by msplapps** [comment]
Faced similar issue but with textfield enabled and used interactive swipe

**Comment by hackiftekhar** [comment]
@msplapps The Interactive swipe issue has been already fixed with v6.0.5

**Comment by danielepantaleone** [comment]
I'm facing this issue also using the objective-c version (grabber the latest version from master branch, v 6.3.0)

**Comment by hackiftekhar** [comment]
Then can you please share demo project @danielepantaleone ?

**Comment by slxl** [comment]
Having the same problem, but the difference is that Im using large titles in my navigation controller + scroll view inside

**Comment by slxl** [comment]
Having the same problem, but the difference is that Im using large titles in my navigation controller + scroll view inside

**Comment by slxl** [comment]
Having the same problem, but the difference is that Im using large titles in my navigation controller + scroll view inside

**Comment by seanooi** [comment]
@hackiftekhar Here's a demo project of what I see with tableView scrolling down when presenting a `UISearchController`. It only happens if I set `navigationController?.navigationBar.isTranslucent = false`.
I've also tried adding the ViewController to `IQKeyboardManager.shared.disabledDistanceHandlingClasses` but it doesn't look like it's doing anything.
[IQKeyboardBug.zip](https://github.com/hackiftekhar/IQKeyboardManager/files/3379512/IQKeyboardBug.zip)
